x/ibc/02-client/client/cli/tx.go

```diff
@@ -74,35 +74,47 @@ $ %s tx ibc client create [client-id] [path/to/consensus_state.json] --from node
 // https://github.com/cosmos/ics/tree/master/spec/ics-002-client-semantics#update
 func GetCmdUpdateClient(cdc *codec.Codec) *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "update [client-id] [path/to/header.json]",
+		Use:   "update [client-id] [path/to/old-header.json] [path/to/new-header.json]",
```

Comment:
Why do we need both the old and the new header?

Reply:
Can't we just read the old header from the store?

Reply:
headers are not stored

Reply:
Shouldn't the headers be stored in the lite client database?

Reply:
@jackzampolin looks like they're only using `lite.Verify` function for verification, nothing else.

Reply:
I also don't see why we need the old header, I think we decided to store the validator set. What are we not storing?

Reply:
This is my question. I don't think we should require the user to pass in multiple headers. This is likely to lead to confusion and will make these lite-client applications much more difficult to use. My question: shouldn't the lite client store all the headers and other consensus state in a `leveldb` backend on disk? Isn't this how the old one worked too? And if we give the lite client a `client-id` it should be able to pull up the historical headers from that chain and use them to populate the `old-header.json` data?